src/main/java/com/happeningnow/Model/Artist.java

```diff
@@ -0,0 +1,25 @@
+package com.happeningnow.Model;
+
+import jakarta.persistence.*;
+import lombok.*;
+import org.hibernate.annotations.UuidGenerator;
+import java.util.List;
+import java.util.UUID;
+
+@Data
```

Comment:
if you are using @Data, then you do not need @AllArgsConstructor and @NoArgsConstructor, I guess

Reply:
ok